recipes/flask-assets/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{%set name = "Flask-Assets" %}
+{%set version = "0.12" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "6031527b89fb3509d1581d932affa5a79dd348cfffb58d0aef99a43461d47847" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+  patches:
```

Comment:
Please add a link to PR ( https://github.com/jeanphix/Ghost.py/issues/311 ) in a comment in the recipe with a brief explanation of its purpose.